tests/conftest.py

```diff
@@ -3,15 +3,25 @@
 import cellprofiler.object
 import cellprofiler.pipeline
 import cellprofiler.workspace
+import numpy
 import skimage.data
 import pytest
 
 
-@pytest.fixture(scope="module")
-def image():
-    example = skimage.data.camera()
-
-    return cellprofiler.image.Image(example)
+@pytest.fixture(scope="module",
+                params=[
+                    (skimage.data.camera()[0:128, 0:128], 2),
+                    (skimage.data.astronaut()[0:128, 0:128, :], 2),
```

Comment:
Nice choice. Typically, I use [mandrill](http://graphics.cs.williams.edu/data/images/mandrill.png). But I like this better.